### PR TITLE
Update 10-dns.sh

### PR DESCRIPTION
**Pull request by a10kiloham**
Clarifying notes for setting IP if you use different IP ranges

### CODE REVIEW

**Review by SturdyErde — CHANGES_REQUESTED** [review]
If I'm not mistaken, this PR includes inline documentation (as noted in the commit message) and also a functional code change. Those should ideally be two separate PRs, with an explanation of the code change's purpose/benefits. Thanks!

File: dns-common/on_boot.d/10-dns.sh [hunk 1]
```diff
@@ -71,7 +75,8 @@ fi
 
 # Make DNSMasq listen to the container network for split horizon or conditional forwarding
 if ! grep -qxF interface=br$VLAN.mac /run/dnsmasq.conf.d/custom.conf; then
-    echo interface=br$VLAN.mac >> /run/dnsmasq.conf.d/custom.conf
```
Comment (SturdyErde) [inline dns-common/on_boot.d/10-dns.sh:74]: Can you expound on the removal of the `echo` command here? (Minor, but the added line break can probably be removed as well.)

Comment (a10kiloham) [reply dns-common/on_boot.d/10-dns.sh:74]: oops, sorry the removal of echo was my own fat fingeredness :)

Comment (SturdyErde) [reply dns-common/on_boot.d/10-dns.sh:74]: Ah, glad I commented. 
+1 for code review! New commit coming? ;)

**Comment by a10kiloham** [comment]
Yes just on mobile now but will fix asap :)

**Pull request closed without merging**